src/i18n/I18nProvider/I18nProvider.tsx

```diff
@@ -0,0 +1,12 @@
+import { Fragment } from 'react';
+import { IntlProvider } from 'react-intl';
+
+import messages from '../messages';
+
+import { TProvider } from './I18nProvider.types';
+
+export const I18nProvider = ({ children, locale }: TProvider) => (
+  <IntlProvider textComponent={Fragment} locale={locale} messages={messages[locale]}>
```

Comment:
After update `TProvider` type you can use rest and spread operators like that:

```suggestion
export const I18nProvider = ({ children, locale, ...props }: TProvider) => (
  <IntlProvider textComponent={Fragment} locale={locale} messages={messages[locale]} {...props} >
```